flink-connector-kudu/src/main/java/org/apache/flink/connectors/kudu/table/KuduTableFactory.java

```diff
@@ -65,6 +65,7 @@
     public static final String KUDU_HASH_COLS = "kudu.hash-columns";
     public static final String KUDU_PRIMARY_KEY_COLS = "kudu.primary-key-columns";
     public static final String KUDU_REPLICAS = "kudu.replicas";
+    public static final String KUDU_IS_BOUNDED = "kudu.is-bounded";
```

Comment:
Why are we introducing the is-bounded flag? As far as I can tell this only affects the isBounded() method of the KuduTableSource and not the actual reading of the data.

The Kudu reading logic at the moment reads the table contents at the start of the job so it is always bounded, I think we should not touch this logic now. 

What do you think?

Reply:
For the `is-bounded` flag, it really just control the `isBounded()` method in 
`KuduTableSource` which inherit from `StreamTableSource`. The main reason I add this is because there may be some users who want to use this kudu connector in a Flink Batch SQL job. And this is usually used under the Blink Batch mode. I also added a few notes to the README file. In addition, I keep the default behavior of this `is-bounded` flag, which is false in `StreamTableSource`.  :)

Reply:
already remove this unnecessary flag